### PR TITLE
DEP Remove unsupported modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/recipe-kitchen-sink/issues/29

### CODE REVIEW

**Review by maxime-rainville — APPROVED** [review]
LGTM

File: phpunit.xml.dist [hunk 1]
```diff
@@ -71,8 +64,6 @@
         <testsuite name="recipe-blog">
             <directory>vendor/silverstripe/blog/tests</directory>
             <directory>vendor/silverstripe/lumberjack/tests</directory>
-            <directory>vendor/silverstripe/widgets/tests</directory>
-            <directory>vendor/silverstripe/content-widget/tests</directory>
             <directory>vendor/silverstripe/spamprotection/tests</directory>
             <directory>vendor/silverstripe/comments/tests</directory>
             <directory>vendor/silverstripe/comment-notifications/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist]: Shouldn't the two comment test suite be removed as well?

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: phpunit.xml.dist (hunk 1 above)
Comment (emteknetnz) [reply phpunit.xml.dist]: yes. updated

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**